### DISCUSSION TITLE
微信播放视频的时候怎么关闭默认全屏播放

### DISCUSSION THREAD

**Discussion by wangzhigang** [bytedance/xgplayer#849] [Q&A] [answered]
**您使用的西瓜播放器版本是多少？ What version of xgplayer are you using?**
2.32.1

**您使用的操作系统和浏览器分别是？ What OS and browser are you using?**
微信

**如何复现问题？ What did you do?**
用微信打卡 H5 地址，播放视频，会默认打开全屏播放

**您期望的播放器正常行为是？ What did you expect to happen?**
不默认全屏播放，在容器里面正常播放

**实际播放器的表现是？ What actually happened?**
点击播放视频按钮，会自动全屏播放视频

**可填写您所在的公司和相关产品业务，方便我们提供更好的技术支持 You can write your company and product which uses xgplayer,  for helping us provide better technical support.**

**Comment by gemxx** [comment, marked as answer]
配置：
`{
  'x5-video-player-fullscreen': false
}`

请参考 https://v2.h5player.bytedance.com/config/#%E5%BE%AE%E4%BF%A1%E5%85%A8%E5%B1%8F%E6%92%AD%E6%94%BE
